.github/cst.yaml

```
@@ -0,0 +1,26 @@
+schemaVersion: '2.0.0'
+
+commandTests:
+  - name: "Ubuntu version"
+    command: ["cat", "/etc/os-release"]
+    expectedOutput: ["Ubuntu 24.04"]
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)